src/js/components/Paragraph.js

```diff
@@ -31,5 +32,6 @@ Paragraph.propTypes = {
   align: PropTypes.oneOf(['start', 'center', 'end']),
   id: PropTypes.string,
   margin: PropTypes.oneOf(['none', 'small', 'medium', 'large']),
-  size: PropTypes.oneOf(['small', 'medium', 'large'])
+  size: PropTypes.oneOf(['small', 'medium', 'large']),
+  width: PropTypes.oneOf(['small', 'medium', 'large'])
```

Comment:
I'm thinking then that we should align widths for other things, like Image, Video, and Quote, to allow pages that combine these elements to achieve a more consistent layout. One option would be to put the width constraint on Box and use full for all of the others, to fill the available width. That would work well for including other things too, like Chart, and Distribution.